src/Shellbar/Shellbar.test.js

```diff
@@ -51,101 +50,6 @@ describe('<Shellbar />', () => {
             profileMenu={profileMenu} />
     );
 
-    const searchInput = {
-        label: 'Search',
-        placeholder: 'Enter a fruit',
-        onSearch: function(searchTerm) {
-            alert(`Search fired for ${searchTerm}`);
-        },
-        callback: () => alert('Search selected!')
-    };
-
-    const actions = [
-        {
-            glyph: 'settings',
-            label: 'Settings',
-            notificationCount: 5,
-            callback: () => alert('Settings selected!'),
-            menu: (
-                <Menu>
-                    <MenuList>
-                        <MenuItem url='/'>Option 1</MenuItem>
-                        <MenuItem url='/'>Option 2</MenuItem>
-                        <MenuItem url='/'>Option 3</MenuItem>
-                    </MenuList>
-                </Menu>
-            )
-        }
-    ];
-
-    const notifications = {
-        notificationCount: 2,
-        label: 'Notifications',
-        notificationsBody: (
-            <Menu>
-                <MenuList>
-                    <MenuItem url='/'>Notification 1</MenuItem>
-                    <MenuItem url='/'>Notification 2</MenuItem>
-                    <MenuItem url='/'>Notification 3</MenuItem>
-                </MenuList>
-            </Menu>
-        ),
-        noNotificationsBody: (
-            <Menu>
-                <MenuList>
-                    <MenuItem>There are no notifications</MenuItem>
-                </MenuList>
-            </Menu>
-        ),
-        callback: () => alert('Notification selected!')
-    };
-
-    const profile = {
-        initials: 'JS',
-        userName: 'John Snow',
-        colorAccent: 8
-    };
-
-    const productMenu = [
-        { name: 'Application A', callback: () => alert('Application A selected!') },
-        { name: 'Application B', callback: () => alert('Application B selected!') },
-        { name: 'Application C', callback: () => alert('Application C selected!') },
-        { name: 'Application D', callback: () => alert('Application D selected!') }
-    ];
-
-    const productSwitcherList = [
-        {
-            title: 'Fiori Home',
-            glyph: 'home',
-            callback: () => alert('Fiori Home selected!')
-        },
-        {
-            title: 'S/4 HANA Cloud',
-            glyph: 'cloud',
-            callback: () => alert('S/4 HANA Cloud selected!')
-        }
-    ];
-
-    const productSwitcher = {
-        label: 'Product Switcher'
-    };
-
-    const coPilotShell = (
-        <Shellbar
-            logoSAP
-            productTitle='Corporate Portal'
-            productMenu={productMenu}
-            subtitle='Subtitle'
-            copilot
-            searchInput={searchInput}
-            actions={actions}
-            notifications={notifications}
-            profile={profile}
-            profileMenu={profileMenu}
-            productSwitcher={productSwitcher}
-            productSwitcherList={productSwitcherList} />
-    );
-
```

Comment:
Were these made with the intention of using them in the snapshot tests?

Reply:
That's a good question.  There were a bunch of variables defined to create it, but then it was never used.  I will check the blame on this to see if someone wants to retain all this code in a separate branch, but it should all be removed from `master`.

Reply:
My bad, i think i forgot to make the snapshot call.  I will create a chore: PR to add the call in there. This snapshot call bumps up the code coverage from 30% to about 85%

Reply:
@chrismanciero Do you want to fix it in this PR or create a separate one?